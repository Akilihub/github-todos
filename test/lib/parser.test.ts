import "jest";
import parseData from "../../src/lib/parser";

const data = {
    "type": "file",
    "encoding": "base64",
    "size": 5362,
    "name": "README.md",
    "path": "README.md",
    "content": `//TODO: @aleku399 look into why this was causing bars not to show`,
};

describe("parsed data tests", () => {
    it.only("should return modified parsed data", () => {
      const _data = parseData(data);
      expect(_data).toMatchSnapshot();
    });
});

