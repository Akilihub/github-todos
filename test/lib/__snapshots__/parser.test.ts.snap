// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parsed data tests should return modified parsed data 1`] = `
Array [
  Object {
    "commentText": "TODO: @aleku399 look into why this was causing bars not to show",
    "fileName": "README.md",
    "keyWord": "TODO:",
    "lineNumber": Object {
      "end": Object {
        "column": 65,
        "line": 1,
      },
      "start": Object {
        "column": 0,
        "line": 1,
      },
    },
  },
]
`;
